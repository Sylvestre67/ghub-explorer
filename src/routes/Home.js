import React from 'react';

import Divider from '@material-ui/core/Divider';

import Form from '../component/Form';
import ResultsProvider from '../component/ResultsProvider';
import Results from '../component/Results';

export class Home extends React.PureComponent {
	render() {
		return (
			<React.Fragment>
				<Form />
				<Divider />
				<ResultsProvider>
					<Results />
				</ResultsProvider>
			</React.Fragment>
		);
	}
}

export default Home;
