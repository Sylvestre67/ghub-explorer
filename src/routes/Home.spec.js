import React from 'react';
import { shallow } from 'enzyme';

import { Home } from './Home';
let component;

const props = {
	classes: {},
};

describe('Home', () => {
	beforeAll(() => {
		component = shallow(<Home {...props} />);
	});

	it('should render correctly', () => {
		expect(component).toMatchSnapshot();
	});
});
