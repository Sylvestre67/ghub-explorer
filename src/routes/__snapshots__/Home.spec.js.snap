// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<Fragment>
  <WithStyles(withRouter(Form)) />
  <WithStyles(Divider) />
  <withRouter(ResultsProvider)>
    <Results />
  </withRouter(ResultsProvider)>
</Fragment>
`;
